src/libraries/Microsoft.Extensions.Configuration.Binder/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
how did you get that whitespace diff? did you use VS to update resx file?

Reply:
That is strange.  I used the latest preview release of Visual Studio 2019.  I'll reapply my change and see if it happens again.  If it does, I'll add it manually via a text editor.

Reply:
I reapplied this change and the differences are now what we'd expect to see.  I don't know what went wrong there!